.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rust-analyzer.cargo.extraEnv": {
+        "LIBCLANG_PATH": "/Library/Developer/CommandLineTools/usr/lib/"
+    }
+}
```

Comment:
I don't know if it's possible, but maybe there is a way to locally configure vscode to prevent this from being added? In a way that we can keep the repo code-editor-agnostic